docs/checks/duplicateRelationCheck.md

```diff
@@ -0,0 +1,16 @@
+# Duplicate Relation Check
+
+#### Description
+The Duplicate Relation check flags Relations in OSM that have the same OSM tags and the same members with same roles.
+we don't want instances of duplicate Relation that represent the same feature. 
+
+#### Live Example
+The following examples illustrate two relations have the same tags and member.
+1) This Relation [id:10392649](https://www.openstreetmap.org/relation/10392649) 
+2) This Relation [id:10392650](https://www.openstreetmap.org/relation/10392650) 
```

Comment:
Do you know approximately what percentage of the flags from this check are like this, where the relations only have one member? I'm wondering how much overlap there might be with what is flagged by OneMemberRelationCheck.

Reply:
@Bentleysb , thanks for your review, I just run for the percentage of the flags where the relations only have one member on OSM: 

> ARG is 11/53,  20%
> AUS is 24/105, 22%
> ISR is 9/19, 47%
> MLT is 2/2, 100%
> NZL is 4/12, 33%
> SWE is 4/14,  28%
> TUR is 12/21, 57%

Reply:
That is quite a bit of overlap, given that each flag effects 2 or more features. I think that have one member is a bigger issue than being a duplicate. Given that, I think it would make sense to not flag one member relations in this check, and leave that for the OneMemberRelaationCheck to give better instructions on how to resolve the larger issue.

Reply:
@Bentleysb , I will exclude one member relations in this check

Reply:
@Bentleysb , could you review again? I already updated for excluding one member relations in this check

Reply:
@Bentleysb ,  I made typo in one of my commits message, I tried to change it, but still show in git log, just wondering if I can close this pull request and make a new PR for this check? so the commit message with typo will not be in git log.

Reply:
@Bentleysb , I tried to change one pushed commit description, but commits got messed up for that, new repository will be more clean for this check, so I closed this one, and open a new PR for this check.